packages/server/src/ai/tools/confluence/index.ts

```diff
@@ -0,0 +1,244 @@
+import { newTool } from "@budibase/types"
+import fetch from "node-fetch"
+import { z } from "zod"
+
+export class ConfluenceClient {
+  private apiToken: string
+  private email: string
+  private baseUrl: string
+
+  constructor(apiToken?: string, email?: string, baseUrl?: string) {
+    this.apiToken = apiToken || process.env.ATLASSIAN_API_TOKEN || ""
+    this.email = email || process.env.ATLASSIAN_EMAIL || ""
+    this.baseUrl =
+      baseUrl ||
+      process.env.ATLASSIAN_BASE_URL ||
```

Comment:
Any reason you're using `process.env` instead of the `env` package we've created? You'd be able to set a default value in there.